src/components/NavAccordion.tsx

```diff
@@ -0,0 +1,60 @@
+import { Icon } from "@canonical/react-components";
+import { FC, ReactNode, useEffect, useState } from "react";
+import { useLocation } from "react-router-dom";
+
+interface Props {
+  baseUrl: string;
+  title: string;
+  children: ReactNode;
+  iconName: string;
+  label: string;
+}
+
+const NavAccordion: FC<Props> = ({
+  baseUrl,
+  title,
+  children,
+  iconName,
+  label,
+}) => {
+  const location = useLocation();
+  const [isOpen, setIsOpen] = useState(false);
+  const isActive = location.pathname.includes(baseUrl);
```

Comment:
```suggestion
  const isActive = location.pathname.includes(baseUrl);
  const [isOpen, setIsOpen] = useState(isActive);
```
This should fix the case the user reloads in a nested page. Then the parent should be open.

Reply:
Aaah yes! Thanks for this